app/views/users/_user.html.erb

```diff
@@ -0,0 +1,9 @@
+<section>
+    <% unless user.Photo.nil? %>
+       <%= image_tag (user.Photo), height: '115', width: '115' %>
+    <% end %>
+    <div>
+      <h2> <%= user.Name %></h2>
+      <span>Number of posts: <%=user.PostsCounter %> </span>
```

Comment:
- [ ] [REQUIRED] Please make sure that PostsCounter display the correct number. Please do it in each page that needs it